Backends/CLX/basic.lisp

```diff
@@ -34,8 +34,16 @@
    (font-families :initform nil :accessor font-families)
    (cursor-table :initform (make-hash-table :test #'eq)
                  :accessor clx-port-cursor-table)
-   (pointer :reader port-pointer)))
-
+   (pointer :reader port-pointer)
+   (font-renderer :initarg :font-renderer
+                  :reader clx-port-font-renderer)))
+
+(defmethod initialize-instance :after ((obj clx-basic-port) &key)
+  (let ((renderer (getf (cdr (port-server-path obj)) :font-renderer)))
```

Comment:
what if path doesn't have a font-renderer?

Reply:
It always does. The default renderer is set up in `PARSE-CLX-SERVER-PATH`. It's set to `CLX-STANDARD-FONT-RENDERER` if none have been supplied.